packages/indexer-agent/src/commands/start.ts

```diff
@@ -194,10 +194,22 @@ export default {
         default: true,
         group: 'Indexer Infrastructure',
       })
-      .option('allocation-claim-threshold', {
-        description: `Minimum query fees (in GRT) received for an allocation to collect and claim them on-chain`,
-        type: 'number',
-        default: 0,
+      .option('rebate-claim-min-single-value', {
```

Comment:
Can we simplify this argument name a bit?  

Maybe `rebate-claim-min` and then its batch counterpart would be `rebate-claim-batch-min`?  Another option I've used here in the past is `threshold` which would make it `rebate-claim-threshold` and `rebate-claim-batch-threshold`.